apps/api/views.py

```diff
@@ -24,22 +26,30 @@ def get_experiments(request):
 
 @api_view(["POST"])
 @permission_classes([HasUserAPIKey])
-def update_participant_data(request):
+def update_participant_data(request, participant_id: UUID):
     """
     Upsert participant data for a specific experiment
     """
-    experiment_public_id = request.data["experiment_id"]
-    identifier = request.data["participant_id"]
-    new_data = request.data["details"]
-    experiment = get_object_or_404(Experiment, public_id=experiment_public_id, team=request.team)
-    participant = get_object_or_404(Participant, identifier=identifier, team=experiment.team)
-
-    try:
-        participant_data = experiment.participant_data.get(participant=participant)
-        participant_data.data = new_data
-        participant_data.save(update_fields=["data"])
-    except ParticipantData.DoesNotExist:
-        ParticipantData.objects.create(
-            participant=participant, data=new_data, content_object=experiment, team=experiment.team
-        )
+    experiment_data = request.data["data"]
```

Comment:
why do we need to wrap this in `data`?

Reply:
Preference I guess. I can change it if that will make things easier

Reply:
I think if the input is already a dict then there's no need to wrap it. If we were using a list then I think wrapping is better.